File: src/targets/node/axios/fixtures/query.js
<<<<<<< HEAD
const axios = require('axios');

const options = {
  method: 'GET',
  url: 'https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value'
};

axios
  .request(options)
  .then(function (response) {
    console.log(response.data);
  })
  .catch(function (error) {
    console.error(error);
  });
=======
const axios = require('axios').default;

const options = {
  method: 'GET',
  url: 'http://mockbin.com/har',
  params: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'}
};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}
>>>>>>> upstream/master

File: src/targets/ruby/faraday/fixtures/query.rb
require 'faraday'

conn = Faraday.new(
  url: 'https://httpbin.org',
)

response = conn.get('/anything') do |req|
  req.params['foo'] = ["bar","baz"]
  req.params['baz'] = 'abc'
  req.params['key'] = 'value'
end

puts response.status
puts response.body